Source/Model/TCCProfile.swift

```diff
@@ -167,8 +167,8 @@ struct TCCProfile : Codable {
                               services: services)
         self.version = 1
         self.uuid = UUID().uuidString
-        self.type = content.type
-        self.scope = "system"
+        self.type = "Configuration"
+        self.scope = "System"
```

Comment:
Do we know if this is correct or not? Looking at [Rich Trouton's repo of example PPPC profiles](https://github.com/rtrouton/privacy_preferences_control_profiles), all of those have `system` lowercase.

Reply:
Looks like the spec has it capitalized, but it might not matter very much. We should stick with the spec though so I think we go with "System"

From the spec:
"Legal values are System and User, with User as the default value."